api/src/test/java/org/openmrs/module/shr/odd/api/impl/test/OnDemandDocumentServiceImplTest.java

```diff
@@ -69,8 +69,6 @@ public void beforeTest() throws Exception
 		Context.getAdministrationService().setGlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_FALSE_CONCEPT, "1066");
         Context.getAdministrationService().setGlobalProperty("shr-cdahandler.cacheMappedConcepts", "false");
 		Context.getAdministrationService().saveGlobalProperty(saveDir);
-		GlobalProperty disableValidation = new GlobalProperty(OpenmrsConstants.GP_DISABLE_VALIDATION, "true");
-		Context.getAdministrationService().saveGlobalProperty(disableValidation);
```

Comment:
This was a global property that we got added to OpenMRS for performance enhancements. Does is not exist in the version that you switched to?